docs/install/Knative-custom-install.md

```diff
@@ -173,6 +173,8 @@ The following Knative installation files are available:
   - https://github.com/knative/serving/releases/download/v0.5.0/monitoring-metrics-prometheus.yaml
   - https://github.com/knative/serving/releases/download/v0.5.0/monitoring-tracing-zipkin.yaml
   - https://github.com/knative/serving/releases/download/v0.5.0/monitoring-tracing-zipkin-in-mem.yaml
+  - https://github.com/knative/serving/releases/download/v0.5.0/monitoring-tracing-jaeger.yaml
+  - https://github.com/knative/serving/releases/download/v0.5.0/monitoring-tracing-jaeger-in-mem.yaml
```

Comment:
There is a table below this list that explains what each of these manifests is. You should update the table as well with short description of what the Jaeger manifests are.

Reply:
Updated, thanks.

Reply:
Great. This is fine except that these manifests don't exist in the release github page. Are they supposed to be there ?

Reply:
No, they would be created as part of the next release as part of https://github.com/knative/serving/pull/3696.

Reply:
then it will be available in the 0.6.0 release and not 0.5.0.

We should tag this for 0.6.0 milestone.

cc/ @samodell @RichieEscarez

Reply:
Do I need to do any changes? I assumed some script would just go through and update all the versions.

Reply:
PRs against master only apply to v.Next. In this case, yes this applies to only v0.6. If it should be added to other versions, you add the cherry-pick label for the corresponding version (although this cherrypicking is a lower priority for the writing team right now until we get caught up).

If a change targets only a specific released version, the PR should be opened against that branch (ie, not master).

Here is a copy of the PR template that tries to address these points:

<-- General PR guidelines:

New contributors:

If you are new to Git/GitHub and want to make a quick fix to the docs,
open your PR against the release branch where you found the error, such as
"release-0.5".

Regular contributors:

Most PRs should be opened against the master branch.

If the change should also be in the most recent numbered release, add the
corresponding "cherrypick-0.X" label; for example, "cherrypick-0.5", to the
original PR. Best practice is to open a PR for the cherry-pick yourself after
your original PR has been merged into the master branch. Once the cherry-pick PR
has merged, remove the cherry-pick label from the original PR.

For more information on contributing to the Knative Docs, see:
https://www.knative.dev/contributing/docs-contributing/

 -->

Reply:
Regarding updating the manifest URLs to v0.6:
Leaving as 0.5 is okay. 

We usually update those paths if and when those files are available at the 0.6 URLs. We are still refining our docs release process but so far, we make a single wide sweeping update across the master branch to change all the URL paths near to the release date.